userdocs/diagnostics/clang-declaration-import.md

```diff
@@ -1,7 +1,6 @@
-# Imported Clang Declaration Warnings (`ClangDeclarationImport`)
+# Imported clang declaration warnings (`ClangDeclarationImport`)
```

Comment:
Clang is a proper noun, so perhaps it should be capitalized?

Reply:
Ah yeah, thanks